packages/wallet/ui/src/util/SuggestChain.js

```diff
@@ -67,12 +67,34 @@ export async function suggestChain(networkConfig, caption = undefined) {
   await window.keplr.enable(chainId);
 
   const offlineSigner = window.getOfflineSigner(chainId);
+  const cosmJS = await SigningStargateClient.connectWithSigner(
```

Comment:
@michaelfig This worked, thanks! Looks like the docs were missing the fee part but I got *something* working below. Unit tests incoming